crates/swc_css_minifier/tests/fixture/compress-calc/vendor/output.min.css

```diff
@@ -0,0 +1 @@
+.class1{width:-webkit-calc(2px)}.class2{width:-WEBKIT-CALC(2px)}.class3{width:-webkit-calc(100% - -webkit-calc(50% + 25px))}.class4{width:-WEBKIT-CALC(1px + -WEBKIT-CALC(2px/2))}.class5{width:-moz-calc(2px)}.class6{width:-webkit-calc(2px)}
```

Comment:
`width:-webkit-calc(100% - -webkit-calc(50% + 25px))` -> `width:-webkit-calc(50% - 25px)`

Reply:
This case is now handled.